api/src/main/java/com/velocitypowered/api/proxy/ProxyServer.java

```diff
@@ -19,7 +20,26 @@
 /**
  * Provides an interface to a Minecraft server proxy.
  */
-public interface ProxyServer {
+public abstract class ProxyServer {
+
+  private static ProxyServer instance;
```

Comment:
We dont want to use static access to ProxyServer, just pass a instance to VelocityChannelIdentifier.